labelbox/schema/model_run.py

```diff
@@ -16,20 +17,40 @@ class ModelRun(DbObject):
     created_by_id = Field.String("created_by_id", "createdBy")
     model_id = Field.String("model_id")
 
-    def upsert_labels(self, label_ids):
+    def upsert_labels(self, label_ids, timeout_seconds=600):
+        """ Calls GraphQL API to start the MEA labels registration process
+        Args:
+            label_ids (list): label ids to insert
+            timeout_seconds (float): Max waiting time, in seconds.
+        Returns:
+            ID of newly generated async task
+        """
 
         if len(label_ids) < 1:
             raise ValueError("Must provide at least one label id")
 
-        query_str = """mutation upsertModelRunLabelsPyApi($modelRunId: ID!, $labelIds : [ID!]!) {
-          upsertModelRunLabels(where : { id : $modelRunId}, data : {labelIds: $labelIds})}
-          """
-        res = self.client.execute(query_str, {
-            'modelRunId': self.uid,
-            'labelIds': label_ids
-        })
-        # TODO: Return a task
-        return True
+        sleep_time = 5
+
+        mutation_name = 'createMEAModelRunLabelRegistrationTask'
+        query_str = """mutation createMEAModelRunLabelRegistrationTaskByApi($modelRunId: ID!, $labelIds : [ID!]!) {
+          %s(where : { id : $modelRunId}, data : {labelIds: $labelIds})}
+          """ (mutation_name)
+
+        while True:
+            res = self.client.execute(query_str, {
```

Comment:
This is going to re-enqueue a new task each time. We should be polling for the status here.